tests/e2e/compatibility/02-assert.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestAssert
-commands:
-- command: /bin/sh -c "kubectl get --namespace $NAMESPACE tempo simplest -o jsonpath='{.status.conditions[?(@.type==\"Ready\")].status}' | grep True"
```

Comment:
Is this condition checked anywhere?

Reply:
Looks like there was a bug in v0.1.3 converter, I will check and fix if needed.
Migrating from one tool to the other automatically is not an easy task 😅

Reply:
Fixed